superbench/benchmarks/micro_benchmarks/nvbandwidth.py

```diff
@@ -111,6 +135,8 @@ def _preprocess(self):
 
         if self._args.disable_affinity:
```

Comment:
Could you please show that "numa support" should work with "--disableAffinity" in somewhere like docs?

Reply:
sure, added in the disable_affinity help